src/utils/fluid_sys.h

```diff
@@ -288,8 +288,11 @@ fluid_atomic_float_get(volatile float *fptr)
 
 /* Threads */
 
+typedef void fluid_thread_return_t;
+#define FLUID_THREAD_RETURN_VALUE
```

Comment:
And consider adding a comment here pointing out why this is needed to avoid that another maintainer optimizes that out in the future.